### PR TITLE
Feature/webumenia 1253

### PR DESCRIPTION
**Pull request by igor-kamil**
# Description

update abandoned composer packages

Fixes [WEBUMENIA-1253](https://jira.sng.sk/browse/WEBUMENIA-1253)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [] I have made corresponding changes to the documentation

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,16 +13,13 @@
     }],
     "require": {
         "laravel/framework": "5.8.*",
-        "caseyamcl/Phpoaipmh": "dev-master",
+        "caseyamcl/phpoaipmh": "dev-master",
         "intervention/image": "dev-master",
         "guzzlehttp/guzzle": "dev-master",
         "elasticsearch/elasticsearch": "^7.2",
         "rtconner/laravel-tagging": "~2.1",
-        "maatwebsite/excel": "~2.1.0",
```
Comment (eronisko) [inline composer.json]: Are we getting rid of this altogether?

File: composer.json [hunk 2]
```diff
@@ -13,16 +13,13 @@
...
         "rap2hpoutre/laravel-log-viewer": "dev-master",
         "zizaco/entrust": "5.2.x-dev",
-        "roumen/sitemap": "2.6.*",
-        "cviebrock/image-validator": "^2.1",
```
Comment (eronisko) [inline composer.json]: Same here -- do we not need this anymore?

File: composer.json (hunk 1 above)
Comment (igor-kamil) [reply composer.json]: yes. we use `league/csv` instead (which support read in more memory efficient way)

File: composer.json (hunk 2 above)
Comment (igor-kamil) [reply composer.json]: we use `laravelium/sitemap` and `illuminate/validation` instead

**Pull request merged**